app/src/store/voice/accessToken.ts

```diff
@@ -34,7 +36,10 @@ export const getAccessToken = createTypedAsyncThunk<
 >('voice/getAccessToken', async (_, { getState, rejectWithValue }) => {
   const user = getState().user;
   if (user?.status !== 'fulfilled') {
-    return rejectWithValue({ reason: 'USER_NOT_FULFILLED' });
+    return rejectWithValue({
+      reason: 'USER_NOT_FULFILLED',
+      error: miniSerializeError(null),
```

Comment:
Do we need this?

Reply:
Without it I have type issues. Might need your help to trouble shoot.